src/controllers/sizes.js

```diff
@@ -0,0 +1,23 @@
+const { SizesService } = require('../services/');
+
+const sizesService = new SizesService();
```

Comment:
Make this as private field of SizesController class (#sizesService = new SizesService())